### ISSUE TITLE
Object Properties always set with tailwind theme.

### ISSUE THREAD

**Issue by Lilwolf** [json-editor/json-editor#609]
### General information

2.x theme tailwind selects all Object Properties, even if the initial value only contains a subset of what's in the schema.  This works fine if I change it to foundation5 (not sure if this is a theme issue or a 2.0 theme issue.  I'll try and test the others.

- json-editor version:

### Expected behavior

When setting an initial value, that is a subset of the schema, I would expect to see only the values set in the initial value selected in the Object Properties... and having to turn them on to add/edit the fields

### Actual behavior

Everything is selected in the Object Properties.

### Steps to reproduce the behavior

Direct link to example: https://json-editor.github.io/json-editor/

```javascript
{
  "title": "json schema example",
  "type": "object",
  "properties": {
    "example": {
      "type": "string",
      "description": "This is an example schema.",
      "default": "Please edit me."
    }
  }
}
```

**Comment by Lilwolf** [comment]
Spectre is working as expected.  Seems like it's tailwind specific..

**Comment by pmk65** [comment]
@mirannda Can you take a look at this?

**Comment by mirannda** [comment]
@pmk65, @Lilwolf 

I see no problems, follow an example

https://is.gd/lh6bay

**Comment by pmk65** [comment]
@mirannda I'm not sure what the problem should be and @Lilwolf haven't posted an test example yet.
But I did notice another 'error' in the Tailwind theme. 
Example: https://is.gd/BayRMr
Click the "Properties" button and then turn off the "Book" property. Then you end up with a small box with vertical scrollbars.
![image](https://user-images.githubusercontent.com/478711/68167847-42dced00-ff67-11e9-8a1b-c2f72478c343.png)

**Comment by Lilwolf** [comment]
Sorry, I had to move on before I could get a simpler case to show.  I'll try and replicate it today.  I found the same when adding a Spectre icon (JSONEditor.defaults.options.iconlib = 'spectre';) but only after having a super complicated schema.  (ie, comment out the iconlib, and everything works as expected, add it and it's gone)... Unfortunately, for work, I'm not working in a full browser, but instead of javafx's webview.  So I can't debug the issue easily.
